api/experimental/api.go

```diff
@@ -57,12 +57,40 @@ func (w Wrapper) VerifySignature(ctx echo.Context) error {
 		return echo.NewHTTPError(http.StatusInternalServerError, fmt.Sprintf("unable to convert the verifiable presentation: %s", err.Error()))
 	}
 
-	validationResult, err := w.Auth.ContractClient().VerifyVP(rawVP)
+	checkTime := time.Now()
```

Comment:
if you want it empty as to fill it later just declare it as

`var checkTime time.Time`

or

 `checkTime := time.Time{}`

Oh I see it might be empty in `requestParams`, is that OK?